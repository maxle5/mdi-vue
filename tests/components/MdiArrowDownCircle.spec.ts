
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiArrowDownCircle from "../../src/components/MdiArrowDownCircle.vue";

test("MdiArrowDownCircle snapshot", () => {
  const wrapper = mount(MdiArrowDownCircle, {});
  expect(wrapper.html()).toMatchSnapshot();
});
