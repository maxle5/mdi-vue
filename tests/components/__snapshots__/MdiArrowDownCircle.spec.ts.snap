// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiArrowDownCircle snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M11,6V14L7.5,10.5L6.08,11.92L12,17.84L17.92,11.92L16.5,10.5L13,14V6H11M12,22A10,10 0 0,1 2,12A10,10 0 0,1 12,2A10,10 0 0,1 22,12A10,10 0 0,1 12,22Z" style="fill: currentColor;"></path>
</svg>"
`;
